lib/redd/models/submission.rb

```diff
@@ -111,7 +111,7 @@ def unlock
       def set_suggested_sort(suggested)
         @client.post(
           '/api/set_suggested_sort',
-          id: get_attribute(:id),
+          id: get_attribute(:fullname),
```

Comment:
Gah, fixed that everywhere else in my code last night and promptly forgot about it until now.